### PR TITLE
Implement `X11` connector name.

### PR DESCRIPTION
**Pull request by TornaxO7**
Closes https://github.com/eneoli/flakeshot/issues/58

### CODE REVIEW

File: src/backend/mod.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ pub type Pixel = u16;
 #[derive(Debug, Clone)]
 pub enum MonitorInfo {
     /// Some additional values in the X11 context.
-    X11 { name: u32 },
+    X11 { output_name: String },
```
Comment (eneoli) [inline src/backend/mod.rs:26]: Can we name it just `name`? It would be equal to the Wayland counterpart then.

**Review by eneoli — CHANGES_REQUESTED** [review]
lgtm. Only one tiny thingy :)

File: src/backend/mod.rs (hunk 1 above)
Comment (TornaxO7) [reply src/backend/mod.rs:26]: ok

Comment (TornaxO7) [reply src/backend/mod.rs:26]: done

File: src/frontend/main_window.rs [hunk 2]
```diff
@@ -87,7 +87,7 @@ fn init_monitor(
     let (monitor, x, y, width, height) = {
         let monitor_name = match &output_info.monitor_info {
             MonitorInfo::Wayland { name, .. } => name,
-            MonitorInfo::X11 { .. } => todo!(), // TODO #58
+            MonitorInfo::X11 { name: output_name } => output_name,
```
Comment (eneoli) [inline src/frontend/main_window.rs:90]: Why not just returning name?

**Review by eneoli — CHANGES_REQUESTED** [review]

File: src/frontend/main_window.rs (hunk 2 above)
Comment (TornaxO7) [reply src/frontend/main_window.rs:90]: bruh, the lsp didn't fix that.

**Review by eneoli — APPROVED** [review]

**Pull request merged**